querybook/server/logic/board.py

```diff
@@ -202,3 +202,19 @@ def get_all_public_boards(environment_id, session=None):
         .filter(Board.environment_id == environment_id)
         .all()
     )
+
+
+@with_session
+def update_board_item_description(board_item, description, session=None):
```

Comment:
just 1 update function is enough, also BoardItem.update(...) already implements everything thats needed 👍 

refer to `update_board(`